src/components/404/NotFound.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import PageLoader from '../PageLoader';
+import NotFoundError from './NotFoundError';
+import Footer from '../Footer';
+
+class NotFound extends React.Component {
```

Comment:
this is a statelless component please change this. if you have eslint set up properly you would see the error